packs/jenkins/README.md

```diff
@@ -0,0 +1,17 @@
+# Jenkins Integration Pack
+
+This pack creates a basic integration with Jenkins
+
+To trigger events from Jenkins, use Jenkins to send a webhook to
+StackStorm. Examples of rules can be found in the `rules` directory.
+
+## Configuration
+
+* `url` - FQDN to Jenkins API endpoint
```

Comment:
Would also be good to clarify the format with an example (e.g. it's a full url or a base url)...